From 709f0a76bea6a62dbc2d6e8fab7ddfcb81a89c9f Mon Sep 17 00:00:00 2001
From: Jingchang Lu <jingchang.lu@freescale.com>
Date: Wed, 30 Apr 2014 22:04:52 +0800
Subject: [PATCH 073/255] arm: dts: ls1021a-qds: updte eTSEC phy-handle
 property

Signed-off-by: Jingchang Lu <jingchang.lu@freescale.com>
[Kevin: Original patch taken from
LS1021A-SDK-V1.1-ARM-SOURCE-20140815-yocto.iso]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 arch/arm/boot/dts/ls1021a-qds.dts | 14 +++++++++-----
 1 file changed, 9 insertions(+), 5 deletions(-)

diff --git a/arch/arm/boot/dts/ls1021a-qds.dts b/arch/arm/boot/dts/ls1021a-qds.dts
index 86eb580..8a19bf3 100644
--- a/arch/arm/boot/dts/ls1021a-qds.dts
+++ b/arch/arm/boot/dts/ls1021a-qds.dts
@@ -98,18 +98,22 @@
 };
 
 &enet0 {
-	phy-handle = <&phy0>;
-	phy-connection-type = "rgmii-id";
+	tbi-handle = <&tbi0>;
+	phy-handle = <&phy1c>;
+	phy-connection-type = "sgmii";
 };
 
 &enet1 {
-	phy-handle = <&phy1>;
-	phy-connection-type = "rgmii-id";
+	tbi-handle = <&tbi0>;
+	phy-handle = <&phy1d>;
+	phy-connection-type = "sgmii";
+	status = "disabled";
 };
 
 &enet2 {
-	phy-handle = <&phy2>;
+	phy-handle = <&phy1>;
 	phy-connection-type = "rgmii-id";
+	status = "disabled";
 };
 
 &ftm0_1 {
-- 
2.0.2

